package.json

```
@@ -60,5 +60,43 @@
 	},
 	"engines": {
 		"node": ">=16"
-	}
+	},
+	"workspaces": [
+		"alves",
+		"balasana",
+		"burnsbury",
+		"blockbase",
+		"brompton",
+		"calm-business",
+		"coutoire",
+		"dalston",
+		"elegant-business",
+		"exford",
+		"friendly-business",
+		"geologist",
+		"hever",
+		"leven",
+		"mayland",
+		"mayland-blocks",
+		"maywood",
+		"modern-business",
+		"morden",
+		"professional-business",
+		"quadrat",
+		"redhill",
+		"rivington",
+		"rockfield",
+		"russell",
+		"seedlet",
+		"seedlet-blocks",
+		"shawburn",
+		"skatepark",
+		"sophisticated-business",
+		"spearhead",
+		"stow",
+		"stratford",
+		"varia",
+		"videomaker",
+		"zoologist"
+	]
```

In order to avoid having to manually add new themes each time they're adding, we could consider doing something like the following (not sure about the naming of the directory):

```suggestion
  "workspaces": [
    "./packages/*",
  ],
```

That would also involve moving around some of the build, deployment, and publishing scripts to reflect the directory change though.

---

Thats right. Defining packages directory is the best approach. 
Considering the fact that, we may not see many changes/additions to this because in the new block themes we avoid adding build dependencies, I left the directory structure as is.

---

Could we use something like this instead, that covers all directories so we wouldn't need to change the directory structure?

```
"workspaces": ["**/*"],
```

I believe the directories that don't have a package.json file would be skipped.

---

It works! 🎉 
```
"workspaces" : "*"
```